### PR TITLE
Added rich snippet function

### PR DESCRIPTION
**Pull request by hxii**
Function will output LD+JSON element to product pages for Google Rich Snippets.

### CODE REVIEW

File: woocommerce-yotpo/wc_yotpo.php [hunk 1]
```diff
@@ -72,7 +72,8 @@ function wc_yotpo_front_end_init() {
 		if($settings['bottom_line_enabled_product']) {	
 			add_action('woocommerce_single_product_summary', 'wc_yotpo_show_buttomline',7);	
 			wp_enqueue_style('yotpoSideBootomLineStylesheet', plugins_url('assets/css/bottom-line.css', __FILE__));
-		}			
+		}
+		add_action('woocommerce_before_single_product', 'wc_yotpo_show_rs', 5);
```
Comment (billykov) [inline woocommerce-yotpo/wc_yotpo.php]: @hxii this will be reflected in the settings page, or enabled by default?

Comment (hxii) [reply woocommerce-yotpo/wc_yotpo.php]: @billykov enabled by default

Comment (billykov) [reply woocommerce-yotpo/wc_yotpo.php]: How can you disable it? only by editing the code?

Comment (hxii) [reply woocommerce-yotpo/wc_yotpo.php]: @billykov correct. Settings switch can be added though.

Comment (hxii) [reply woocommerce-yotpo/wc_yotpo.php]: @billykov checkbox was added under the admin page. It is disabled by default.

**Comment by billykov** [comment]
Amazing, thanks!

**Pull request closed without merging**